kroxylicious-filters/kroxylicious-encryption/src/main/java/io/kroxylicious/filter/encryption/EnvelopeEncryption.java

```diff
@@ -61,33 +62,41 @@ record KmsCacheConfig(
 
     }
 
+    public record SharedServices<K, E>(
+                                       Kms<K, E> kms,
+                                       EnvelopeEncryption.Config configuration,
```

Comment:
Given its embedding the config model as well maybe it should be called the `EncryptionFilterContext` ?